docker/test/util/Dockerfile

```
@@ -29,6 +29,9 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /var/cache/debconf /tmp/*
 
+# for external_symbolizer_path
+RUN ln -s /usr/bin/llvm-symbolizer-${LLVM_VERSION} /usr/bin/llvm-symbolizer
+
```

hmmm... ok, is it really needed in test-utils? If so, we rather need to move it into the same RUN where it's installed. Otherwise, let's move all the llvm-related staff into fasttests. I am not sure if it's used in tests

```suggestion
    && ln -s /usr/bin/llvm-symbolizer-${LLVM_VERSION} /usr/bin/llvm-symbolizer \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* /var/cache/debconf /tmp/*

```

And please, comment before the RUN, where the toolchain is used

---

> hmmm... ok, is it really needed in test-utils?

It is needed at least in all tests to make symbolization of stacktraces of sanitizers works, and this is the most basic image.

Though I can move it into the test-base, since it already has other sanitizer settings as well, will do.

> If so, we rather need to move it into the same RUN where it's installed.

It is not installed here, it is installed only in fasttest.

So, what I missed is the llvm-$VER package, fixed.